internal/guestpath/paths.go

```diff
@@ -0,0 +1,57 @@
+package guestpath
```

Comment:
Maybe put this at `internal/guest/path` rather than `internal/guestpath`? Since it seems entirely about in-guest details.

Reply:
I put this like this in anticipation that this PR https://github.com/microsoft/hcsshim/pull/1258 will get merged. The values are reused in both guest and shim now.